### PR TITLE
fixed -M

### PR DESCRIPTION
**Pull request by mtezzele**

### CODE REVIEW

File: include/cxx_project.sh [hunk 1]
```diff
@@ -67,9 +67,8 @@ function make_project
 	else
 		if [ "$4" == "true" ]
 		then
-			rm -rf "$PROJECT_NAME"
+			rm -rf "$PROJECT_NAME/*"
```
Comment (ESeNonFossiIo) [inline include/cxx_project.sh:70]: I do not think this is necessary. -r delete all files contained in the folder

File: include/cxx_project.sh [hunk 2]
```diff
@@ -67,9 +67,8 @@ function make_project
...
 		fi
-		mkdir "$PROJECT_NAME"
```
Comment (ESeNonFossiIo) [inline include/cxx_project.sh:72]: This is needed. You destroy the project and then you want to rebuilt it again.

Comment (mtezzele) [inline include/cxx_project.sh:72]: I just do not want to destroy the folder, just its content. So I do not need to recreate the directory.

Comment (ESeNonFossiIo) [inline include/cxx_project.sh:72]: ok. you are right!

**Pull request merged**